Shared/Models/Device.cs

```diff
@@ -69,6 +69,9 @@ public class Device
         public int ProcessorCount { get; set; }
 
         public string PublicIP { get; set; }
+
+        public string MACAddresses { get; set; }
```

Comment:
Property names should be `PascalCase`.

Also, this should be a `string[]` or similar collection, with a conversion to JSON in `AppDb.OnModelCreating`.